scripts/poppunk_batch_mst.py

```diff
@@ -51,7 +51,10 @@ def get_options():
     aGroup.add_argument('--threads', help='Number of threads for parallelisation (int)',
                                                 type = int,
                                                 default = 1)
-    aGroup.add_argument('--use-gpu', help='Use GPU for analysis',
+    aGroup.add_argument('--gpu-dist', help='Use GPU for distance calculations',
```

Comment:
Starting to feel like a `--gpu` option which takes options sketch, distance, graph or all would be sensible.

Reply:
`dbFuncs` makes this a bit tricky - in particular, the behaviour of trying GPU sketching, then retrying with CPU if assemblies present. Not sure if worth changing all this structure at once, or how it might interface with the web version?